django_admin_keyboard_shortcuts/templates/admin/base.html

```diff
@@ -15,7 +15,7 @@
   <button id="open-shortcuts" data-keyboard-shortcut="?">
     <kbd>?</kbd>
   </button>
-  <dialog id="model-list-dialog">
+  <dialog class="keyboard-shortcut-dialog" id="model-list-dialog">
```

Comment:
The `-dialog` part feels redundant and I think we want the plural form

```suggestion
  <dialog class="keyboard-shortcuts" id="model-list-dialog">
```